src/main/java/com/netflix/imflibrary/st2067_2/ApplicationCompositionFactory.java

```diff
@@ -50,10 +50,15 @@
         add("http://www.smpte-ra.org/ns/2067-50/2017");
     }});
 
+    private static final Set<String> namespacesApplicationTsp2121Composition = Collections.unmodifiableSet(new HashSet<String>() {{
+        add("http://www.digitalproductionpartnership.co.uk/schema/imf/TSP2121-1/2018");
+    }});
+
     public enum ApplicationCompositionType {
         APPLICATION_2_COMPOSITION_TYPE(Application2Composition.class,          namespacesApplication2Composition),
         APPLICATION_2E_COMPOSITION_TYPE(Application2ExtendedComposition.class, namespacesApplication2EComposition),
         APPLICATION_5_COMPOSITION_TYPE(Application5Composition.class,          namespacesApplication5Composition),
+        APPLICATION_TSP_2121_COMPOSITION_TYPE(ApplicationTsp2121Composition.class, namespacesApplicationTsp2121Composition),
```

Comment:
TSP 2121 has many parts. ApplicationTsp2121 could be replaced by ApplicationTsp2121Part1 or ApplicationDPPProRes.